.vscode/launch.json

```diff
@@ -15,7 +15,7 @@
     },
     {
       "name": "Run Web Extension in VS Code",
-      "type": "pwa-extensionHost",
+      "type": "extensionHost",
```

Comment:
I was wondering if you can show a link if it's not required anymore? In their documentation I can still find a reference where they are using it. That's why I'm a little bit confused now:
https://code.visualstudio.com/api/extension-guides/web-extensions#test-your-web-extension-in-vs-code-running-on-desktop

Reply:
For posterity:
![25-01-28 10-06-16 Code](https://github.com/user-attachments/assets/d94289ff-672e-4583-a034-5fafdd1af8e7)